src/js/themes/base.js

```diff
@@ -736,12 +736,12 @@ export const generate = (baseSpacing = 24, scale = 6) => {
     formField: {
       border: {
         color: 'border',
-        error: {
-          color: {
-            dark: 'white',
-            light: 'status-critical',
-          },
-        },
+        // error: {
```

Comment:
Can we add a comment that this will be deprectaed and one should use  `error.border` instead?

Reply:
Yes, I can add that. Thank you for the comments!